skorch/tests/test_classifier.py

```diff
@@ -240,3 +240,43 @@ def test_default_loss_does_call_sigmoid(
 
         net.predict_proba(X)
         assert mock.call_count > 0
+
+    def test_with_calibrated_classifier_cv(self, net_fit, data):
+        from sklearn.calibration import CalibratedClassifierCV
+        cccv = CalibratedClassifierCV(net_fit, cv=3)
```

Comment:
Do we need the fitted module here?

If we care alot about test run time speed, we can set `cv=2`?

Reply:
Changed to `cv=2`. Net has to be initialized for this to work.